contents/docs/contribute/contribute-to-website.md

```diff
@@ -76,7 +74,7 @@ Once you have Visual Studio Code open, select the **Terminal** menu option. With
 
 ![Visual Studio Code terminal](../../images/docs/contribute/visual-studio-code-terminal.png)
 
-Don't worry! We only need to run a few more commands in the terminal.
+Don't worry! We only need to run a few commands in the terminal.
```

Comment:
At this point the person using GitHub Desktop + Visual Studio code hasn't had to do anything in the terminal.

Reply:
Yep, good point. Me being a dumbass.